src/style/components/list/participant-item.less

```diff
@@ -120,7 +118,6 @@
       &__text {
         display: flex;
         min-width: 0; // this will ensure that ellipses is working
-        max-width: @participant-max-width;
         min-height: @avatar-diameter-m;
         flex-direction: column;
         flex-grow: 1;
```

Comment:
Are we sure these style changes don't affect other "types" of participant items we're creating?